src/routes/api/profile.js

```diff
@@ -0,0 +1,18 @@
+import { Router } from 'express';
+import ProfileController from '../../controllers/profileController';
+import middleware from '../../middlewares';
+
+const { Authenticate, validate } = middleware;
+
+const router = Router();
+
+router.route('/')
+  .post(Authenticate.verifyToken, ProfileController.upsertProfile)
+  .get(Authenticate.verifyToken, Authenticate.verifyTravelAdmin, ProfileController.getAllProfiles);
+router.route('/:userId')
+  .get(Authenticate.verifyToken, validate('checkUserId'), Authenticate.verifyOwnership, ProfileController.getProfile)
+  .put(Authenticate.verifyToken, validate('checkUserId'), Authenticate.verifyOwnership, ProfileController.upsertProfile);
+router.route('/roles/:userId')
+  .put(Authenticate.verifyToken, validate('checkUserId'), Authenticate.verifyTravelAdmin, ProfileController.updateRole);
```

Comment:
Why are you giving the `TravelAdmin` the right to assign roles?

Reply:
The same reason I specified below. I will go back and create it's own function separately.